src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,39 @@
 package core.mate.academy.service;
 
-/**
- * Your implementation of MachineService.
- */
-public class MachineServiceImpl {
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.Collections;
+import java.util.List;
+
+public class MachineServiceImpl implements MachineService<Machine> {
+    @Override
+    public List getAll(Class<? extends Machine> type) {
+        if (type.equals(Bulldozer.class)) {
+            BulldozerProducer bulldozerProducer = new BulldozerProducer();
+            return bulldozerProducer.get();
```

Comment:
```suggestion
            return new BulldozerProducer().get();
```
do the same with other if statements